core/thread/thread_pool.odin

```diff
@@ -25,32 +25,28 @@ Task :: struct {
 // Careless access can and will lead to nasty bugs. Once initialized, the
 // pool's memory address is not allowed to change until it is destroyed.
 Pool :: struct {
-	allocator:     mem.Allocator,
-	mutex:         sync.Mutex,
-	sem_available: sync.Sema,
+	allocator:         mem.Allocator,
+	mutex:             sync.Mutex,
+	sem_available:     sync.Sema,
 
 	// the following values are atomic
 	num_waiting:       int,
 	num_in_processing: int,
 	num_outstanding:   int, // num_waiting + num_in_processing
 	num_done:          int,
 	// end of atomics
-
-	is_running: bool,
-
-	threads: []^Thread,
-
-
-	tasks:      queue.Queue(Task),
-	tasks_done: [dynamic]Task,
+	is_running:        bool,
+	threads:           []^Thread,
+	tasks:             queue.Queue(Task),
+	tasks_done:        [dynamic]Task,
```

Comment:
Why have you changed the styling of this file?

Reply:
ols did a number on me. I'll revert the styling on the rest of the file